docs/background/numbering.md

```diff
@@ -61,14 +61,14 @@ amino acid numbering is p.1, p.2, p.3, ..., etc. from the first to the last amin
+    * **c.79<span class="spotlight">-</span>45**G>T is a variant in an intron (3' half): since nucleotide "c.79" is followed by `-` and a second number ("45") the nucleotide is in an intron, 5' of the splice acceptor site and in the 3' half of the intron
+    * **c.<span class="spotlight">-</span>106<span class="spotlight">+</span>**2T>A is a variant in an intron in the 5'UTR (upstream of the translation initiation site): since nucleotide "c.-106" has a `-` prefix it is located 5' of the ATG translation initiation codon and since it is followed by `+` and a second number ("2") the nucleotide is in the 5' half of an intron
+    * **c.<span class="spotlight">\*</span>639<span class="spotlight">-</span>**1G>A is a variant in an intron in the 3'UTR (downstream of the translation termination site): since nucleotide "c.*639" has a `*` prefix it is located 3' of the translation termination codon and since it is followed by `-` and a second number ("1") the nucleotide is in the 3' half of the intron
 
-* When I retrieve a coding DNA sequence from GenBank (NM_ file) nucleotide numbering does not start with 1 at the A of the ATG translation initiation codon.: Correct, but it is not difficult to obtain such a file. Go to the NM_ sequence of interest using [the Nucleotide database](http://www.ncbi.nlm.nih.gov/nucleotide/), e.g. NM\_004006.2. The [page retrieved](http://www.ncbi.nlm.nih.gov/nucleotide/NM_004006.2) is annotated extensively. Clicking the "**CDS**" annotation (CoDing Sequence) opens a window where the nucleotide numbering will start with 1 at the A of the ATG translation initiation (start) codon.<br>: To assist those studying or reporting sequence variants, gene variant databases (LSDBs, see the [HGVS list of LSDBs](http://www.LOVD.nl/LSDBS)) usually provide a coding DNA reference sequence with the nucleotide numbering displayed (see [e.g. the DMD example](http://www.DMD.nl/nmdb2/refseq/DMD_codingDNA.html))
+* When I retrieve a coding DNA sequence from GenBank (NM_ file) nucleotide numbering does not start with 1 at the A of the ATG translation initiation codon.: Correct, but it is not difficult to obtain such a file. Go to the NM_ sequence of interest using [the Nucleotide database](http://www.ncbi.nlm.nih.gov/nucleotide/), e.g. NM\_004006.2. The [page retrieved](http://www.ncbi.nlm.nih.gov/nucleotide/NM_004006.2) is annotated extensively. Clicking the "**CDS**" annotation (CoDing Sequence) opens a window where the nucleotide numbering will start with 1 at the A of the ATG translation initiation (start) codon. A: To assist those studying or reporting sequence variants, gene variant databases (LSDBs, see the [HGVS list of LSDBs](http://www.LOVD.nl/LSDBS)) usually provide a coding DNA reference sequence with the nucleotide numbering displayed (see [e.g. the DMD example](http://www.DMD.nl/nmdb2/refseq/DMD_codingDNA.html))
```

Comment:
Yes, there are all kinds of inconsistencies throughout the text.  I recommend that we make an issue for stuff like this and deal with it as a single PR.  I'd like to decouple changes wherever possible so that's clear what we're fixing in each PR.

Reply:
That's a good idea. Shall we deal with those in PRs that will merge into this branch? That way, they are handled separately, _and_ this branch still gets fixed before it's merged into `main`.